resolwe/process/tests/test_python_process.py

```diff
@@ -389,6 +389,22 @@ def test_create_data(self):
         Collection.objects.get(name=collection_name)
         Data.objects.get(name=data_name)
 
+    @with_docker_executor
+    @tag_process("assign-entity-tags")
+    def test_assign_tags_entity(self):
+        """Assign tags to entity."""
+        data = self.run_process(
```

Comment:
It would be nice to create another entity without edit permission and see what happens if the process tries to change it. Ideally, the listener should return an error message and a verbose error should be raised inside the process.